tests/test_admin_cli.py

```diff
@@ -1123,6 +1144,32 @@ def test_do_export_reference_data(self, patched_input):
         if os.path.exists(path):
             os.remove(path)
 
+    @patch("pepys_admin.snapshot_cli.input")
+    def test_do_export_reference_data_invalid_filename(self, patched_input):
+        with open("already_existing_file.db", "w") as f:
+            f.write("Hello, world")
+
+        patched_input.side_effect = ["already_existing_file.db", "test.db"]
+        temp_output = StringIO()
+        with redirect_stdout(temp_output):
+            self.shell.do_export_reference_data()
+        output = temp_output.getvalue()
+        assert "Reference tables are successfully exported!" in output
```

Comment:
I think it would be better if we also assert the error message of `already_existing_file.db`